content/en/logs/guide/access-your-log-data-programmatically.md

```diff
@@ -0,0 +1,524 @@
+    "query": "*"
+  }
+}'
+
+```
+
+**Response:**
+
+The result dataset is comprised of the `data` object, as depicted in the following example response.
+
+```json
+
+{
+    "meta": {
+        "page": {
+            "after": "eyJhZnRlciI6IkFRQUFBWFVBWFJZVkpHeHZEUUFBQUFCQldGVkJXRkpaVm1kMlpsa3RiVWRVWmpSQlFRIn0"
+        }
+    },
+    "data": [
+        {
+            "attributes": {
+                "status": "info",
+                "service": "pageViewService",
+                "tags": [
+                    "source:postman",
+                    "project:test"
+                ],
+                "timestamp": "2020-10-07T00:02:33.461Z",
+                "host": "my.sample.host",
+                "attributes": {
+                    "hostname": "my.sample.host",
+                    "pageViews": "450",
+                    "user": "chris",
+                    "service": "pageViewService"
+                },
+                "message": "Sample message"
+            },
+            "type": "log",
+            "id": "AQAAAXUAXd91M9wyTgAAAABBWFVBWGQ5MVZrQmFzdEZ2TG9BQQ"
+        },
+        {
+            "attributes": {
+                "status": "info",
+                "service": "pageViewService",
+                "tags": [
+                    "source:postman",
+                    "project:test"
+                ],
+                "timestamp": "2020-10-07T00:01:57.586Z",
+                "host": "my.sample.host",
+                "attributes": {
+                    "hostname": "my.sample.host",
+                    "pageViews": "500",
+                    "user": "bob",
+                    "service": "pageViewService"
+                },
+                "message": "Sample message"
+            },
+            "type": "log",
+            "id": "AQAAAXUAXVNSvuMvWwAAAABBWFVBWFZOU2I2ZWcxX3c2LVVBQQ"
+        },
+        {
+            "attributes": {
+                "status": "info",
+                "service": "pageViewService",
+                "tags": [
+                    "source:postman",
+                    "project:test"
+                ],
+                "timestamp": "2020-10-07T00:01:41.909Z",
+                "host": "my.sample.host",
+                "attributes": {
+                    "hostname": "my.sample.host",
+                    "pageViews": "700",
+                    "user": "steve",
+                    "service": "pageViewService"
+                },
+                "message": "Sample message"
+            },
+            "type": "log",
+            "id": "AQAAAXUAXRYVJGxvDQAAAABBWFVBWFJZVmd2ZlktbUdUZjRBQQ"
+        }
+    ],
+    "links": {
+        "next": "https://api.datadoghq.com/api/v2/logs/events?filter%5Bquery%5D=%2A&page%5Bcursor%5D=eyJhZnRlciI6IkFRQUFBWFVBWFJZVkpHeHZEUUFBQUFCQldGVkJXRkpaVm1kMlpsa3RiVWRVWmpSQlFRIn0&page%5Blimit%5D=3&filter%5Bfrom%5D=2020-10-07T00%3A00%3A00%2B00%3A00&filter%5Bto%5D=2020-10-07T00%3A15%3A00%2B00%3A00"
+    }
+}
+
+```
+
+### Sort by facet or timestamp
+
```

Comment:
for the sake of clarity, could we include here a simple screenshot of this feature in the UI ?